Application Lifecycle Management/api-boxes.md

```diff
@@ -12,58 +13,83 @@ Manage and perform actions on boxes.
 
 | Resource | Description |
 |----------|-------------|
-| GET /services/boxes | Gets the list of boxes that are accessible in the personal workspace. |
-| POST /services/boxes | Creates a new box. |
+| [GET /services/boxes](#get-servicesboxes) | Gets the list of boxes that are accessible in the personal workspace. |
+| [POST /services/boxes](#post-servicesboxes) | Creates a new box. |
 
 **Perform Box Operations**
 
 | Resource | Description |
 |----------|-------------|
-| GET /services/boxes/{box_id} | Fetches an existing box. |
-| PUT /services/boxes/{box_id} | Updates an existing box. |
-| DELETE /services/box/{box_id} | Deletes an existing box. |
-| GET /services/boxes/{box_id}/stack | Gets the box stack. |
-| GET /services/boxes/{box_id}/bindings | Gets the box bindings. |
-| GET /services/boxes/{box_id}/versions | Get the list of box versions. |
-| PUT /services/boxes/{box_id}/diff  | Get the diff between two boxes. |
+| [GET /services/boxes/{box_id}](#get-servicesboxesbox_id) | Fetches an existing box. |
+| [PUT /services/boxes/{box_id}](#put-servicesboxesbox_id) | Updates an existing box. |
+| [DELETE /services/box/{box_id}](#delete-servicesboxesbox_id) | Deletes an existing box. |
+| [GET /services/boxes/{box_id}/stack](#get-servicesboxesbox_idstack) | Gets the box stack. |
+| [GET /services/boxes/{box_id}/bindings](#get-servicesboxesbox_idbindings) | Gets the box bindings. |
+| [GET /services/boxes/{box_id}/versions](#get-servicesboxesbox_idversions) | Get the list of box versions. |
+| [PUT /services/boxes/{box_id}/diff](#put-servicesboxesbox_iddiff)  | Get the diff between two boxes. |
 
 **CloudFormation Box**
 
 The Cloud Application Manager CloudFormation box runs on the AWS CloudFormation service. It lets you create and customize templates that you can launch as a single stack of combined services in AWS. Manage CloudFormation configurations in Cloud Application Manager using these [API actions](./api-boxes.md).
 
-### GET /services/boxes
+Some examples include:
+| Example |
+|----------|
+|[Create a CloudFormation box with template](#example-create-a-CloudFormation-box-with-template)|
+|[Modify the CloudFormation Template](#example-modify-the-cloudFormation-template)|
+|[Launch a CloudFormation Box](#example-launch-a-cloudFormation-box)|
+|[Update a CloudFormation Stack in Real-Time](#example-update-a-cloudFormation-stack-in-real-time)|
 
+
```

Comment:
Please, leave a space between | and first character in a column.
The same for last character and | in a column.
Otherwise, the table is not displayed correctly.